spec.html

```diff
@@ -7459,9 +7456,9 @@ <h1>FunctionCreate ( _kind_, _ParameterList_, _Body_, _Scope_ [ , _prototype_ ]
       <emu-alg>
         1. If _prototype_ is not present, then
           1. Set _prototype_ to the intrinsic object %FunctionPrototype%.
-        1. If _kind_ is not ~Normal~, let _allocKind_ be `"non-constructor"`.
-        1. Else, let _allocKind_ be `"normal"`.
-        1. Let _F_ be FunctionAllocate(_prototype_, _allocKind_).
+        1. If _kind_ is not ~Normal~, let _needsConstruct_ be *false*.
```

Comment:
Can we flip this to get rid of double negatives?

```
If _kind_ is ~Normal~, let _needsConstruct_ be *true*.
Else let _needsConstruct_ be *false*.
```

Reply:
Added a commit for this.